lib/hydra/works/models/concerns/generic_file_behavior.rb

```diff
@@ -0,0 +1,44 @@
+module Hydra::Works
+  module GenericFileBehavior
```

Comment:
That's not a reason to add "Generic" into the name of a module that's not actually Generic.